src/safeds/ml/classical/regression/_lasso_regression.py

```diff
@@ -13,9 +14,25 @@
 
 
 class LassoRegression(Regressor):
-    """Lasso regression."""
-
-    def __init__(self) -> None:
+    """Lasso regression.
+
+    Parameters
+    ----------
+    alpha : float, default=1.0
+        Constant that multiplies the L1 term. Defaults to 1.0.
```

Comment:
```suggestion
    alpha : float
        Controls the regularization of the model. The higher the value, the more regularized it becomes.
```

This does not rely on the mathematical definition of lasso regression as much. If we add the formular in the future, we can add your explanation again.

We usually don't need to repeat the default value here.